django_auth_ldap/backend.py

```diff
@@ -190,6 +193,41 @@ def get_group_permissions(self, user, obj=None):
 
         return permissions
 
+    def get_user_permissions(self, user, obj=None):
+        if not hasattr(user, "ldap_user") and self.settings.AUTHORIZE_ALL_USERS:
+            _LDAPUser(self, user=user)  # This sets user.ldap_user
+
+        if hasattr(user, "ldap_user"):
+            permissions = self._get_permissions(user, obj, "user")
```

Comment:
```suggestion
            permissions = self._get_user_permissions(user, obj, "user")
```

Reply:
Since "user" is hardcoded, why not directly hardcode it instead of adding a `from_name` argument to `_get_permissions`?